### PR TITLE
koa2 compatible

### LINKED ISSUE #3: koa@2 support

### PR DESCRIPTION
**Pull request by PredokMiF**
Commit for compatible with Koa2 fixes #3

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,64 @@
-npm-debug.log
+package-lock.json
+
+### Node template
+# Logs
+logs
+*.log
+npm-debug.log*
+yarn-debug.log*
```
Comment (haoxins) [inline .gitignore:8]: no need so much ignore files :)

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "koa-no-cache",
-  "version": "1.1.0",
+  "version": "2.0.0",
```
Comment (haoxins) [inline package.json]: don't change version in a PR :)

File: .gitignore (hunk 1 above)
Comment (PredokMiF) [reply .gitignore:8]: This is template of Ignore plugin in JetBrains Idea ))

**Comment by haoxins** [comment]
@PredokMiF thanks for the PR ! Please rebase with master branch, then, CI should be passed :)

**Comment by PredokMiF** [comment]
Should I do somethin else to apply this PR?

**Comment by haoxins** [comment]
land 931ab907258b5d83695bade1c6aa3883b7764321

**Comment by haoxins** [comment]
`v2.0.0` published :)

thx

**Comment by PredokMiF** [comment]
Незачто )

**Pull request closed without merging**